spec/ransack/adapters/active_record/base_spec.rb

```diff
@@ -336,6 +336,12 @@ def self.sane_adapter?
           end
         end
 
+        describe '#ransack_alias' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.